hover_preview.py

```diff
@@ -108,7 +107,7 @@ def magick(inp, out):
     subprocess.call(["magick", inp, out], shell=os.name == "nt")
 
 
-def get_dimensions(view: View, path: str) -> (int, int):
```

Comment:
Why remove the return types from functions?

Reply:
Because their syntax makes mypy unhappy. For this specific line, `(int, int)` should be replaced by `Tuple[int, int]` but in this case the plugin will require the typing module as it is not included in python 3.3.